anonym-ip_test.go

```diff
@@ -0,0 +1,341 @@
+package anonymization
+
+import (
+	"bytes"
+	"net"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	// initialize whatever needs to be initialized
+	os.Exit(m.Run())
+}
+
+func TestEncryptIP(t *testing.T) {
+	// set-up if needed
+	t.Run("IPv4, passphrase", func(t *testing.T) {
+		cases := []net.IP{
+			[]byte{1, 2, 3, 4},
+			[]byte{198, 41, 56, 22},
+			[]byte{22, 11, 33, 44},
+			[]byte{255, 0, 255, 241},
+		}
```

Comment:
Possible addition: random IP & random key (e.g. generate 1000 random IP and test encrypt /decrypt with random key/password).